test/test_connected_components.py

```diff
@@ -1,45 +0,0 @@
-import ursa
```

Comment:
I would like to keep this file, at some point I will push a faster implementation for connected components, and it will be nice to already have the test.

Reply:
Oh my bad, putting it back